README.md

```diff
@@ -35,9 +36,12 @@ import InstagramEmbed from 'react-instagram-embed';
 />
 ```
 
+Access token is combination of App Id and Client Token. See https://developers.facebook.com/docs/instagram/oembed/#access-tokens for more details.
+
 ## props
 
 - `url` {String} Instagram URL. Required
+- `accessToken` {String} Instagram Client Access Token. Required
```

Comment:
I think more bettter to `clientAccessToken` naming

Reply:
I agree, clientAccessToken is more specific and makes it clear that appAccessToken should not be used on client-side.  
Fixed.